reservoir-backend.js

```diff
@@ -103,6 +115,9 @@ function ReservoirBackend(options) {
     self.dropCount = {};
     self.logCount = {};
     self.samplingDecision = null;
+
+    var keys = Object.keys(options);
```

Comment:
sure; just implement it in a decent away.

aka `subset(Object.keys(options), WHITELIST)`